.github/workflows/ci.yml

```
@@ -129,27 +132,49 @@ jobs:
+        run: |
+          # Regenerate CI goldens on Linux so pixel comparison is deterministic.
+          # Exit 79 = no tests matched the tag (no golden tests in this package).
+          set +e
+          flutter test --dart-define=CI=true --update-goldens --tags golden
+          rc=$?
+          set -e
+          if [ "$rc" -ne 0 ] && [ "$rc" -ne 79 ]; then exit "$rc"; fi
+        working-directory: ${{ matrix.directory }}
+      - name: Commit Updated Goldens
+        if: ${{ needs.changes.outputs[matrix.change_key] == 'true' }}
+        run: |
```

actionlint (1.7.11)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)